bin/storm.py

```diff
@@ -485,6 +485,11 @@ def kill_workers(*args):
         extrajars=[USER_CONF_DIR, os.path.join(STORM_DIR, "bin")])
 
 def shell(resourcesdir, command, *args):
+    """Syntax: [storm shell resourcesdir command args]
+
+    Makes constructing jar and uploading to nimbus for using non JVM languages
```

Comment:
I feel this line misses behavior "executing command".

I'd like to have this line changed like 
`Archives resources to jar and uploads jar to Nimbus, and executes following arguments on "local". Useful for non JVM languages.`